examples/platform/silabs/FreeRTOSConfig.h

```diff
@@ -107,11 +107,15 @@ extern "C" {
 
 #include <stdint.h>
 
+#ifdef SIWX_917
+extern uint32_t SystemCoreClock;
+#else // For EFR32
```

Comment:
The else is probably not needed. Also why SystemCoreClock needs to be `extern` ?